server/api/events/event.get.ts

```diff
@@ -1,13 +1,9 @@
-import { PrismaClient } from "@prisma/client";
-
 export default defineEventHandler(async event => {
-  const query = getQuery(event)
-  const prisma = new PrismaClient();
-  const records = await prisma.event.findUnique({
-    where:{
+  const query = getQuery(event);
+  const records = await event.context.prisma.event.findUnique({
```

Comment:
similar to above, can't you just return it rather than awaiting, storing, then returning?